src/login/login_errors.h

```diff
@@ -0,0 +1,35 @@
+﻿/*
+===========================================================================
+
+Copyright (c) 2023 LandSandBoat Dev Teams
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see http://www.gnu.org/licenses/
+
+===========================================================================
+*/
+
+#pragma once
+
+namespace loginErrors
+{
+    enum errorCode : uint16
+    {
+        CHARACTER_NAME_UNAVAILABLE              = 313, // "The character name you entered is unavailable.\nPlease choose another name." // This message is displayed in Japanese, needs fixing.
```

Comment:
Can you make this comment a TODO, and maybe a little bit more prominent - on the line above?

Reply:
Done